fix/upp/postxconfig-NT-rrfs_subh.txt

```diff
@@ -1,5 +1,5 @@
 1
-41
+39
```

Comment:
Is the change from 41 to 39 related to the number of records in the file (removing the two column integrated fields)?

Reply:
Yes, that is correct.